code/datums/martial/krav_maga.dm

```diff
@@ -1,189 +1,148 @@
+	if(target.stat || target.IsParalyzed() || target.IsKnockdown())
+		return FALSE
+	var/obj/item/bodypart/affecting = target.get_bodypart(BODY_ZONE_CHEST)
+	var/armor_block = target.run_armor_check(affecting, MELEE)
+	target.visible_message(span_warning("[user] leg sweeps [target]!"), \
+					span_userdanger("Your legs are sweeped by [user]!"), span_hear("You hear a sickening sound of flesh hitting flesh!"), null, user)
+	to_chat(user, span_danger("You leg sweep [target]!"))
+	playsound(get_turf(user), 'sound/effects/hit_kick.ogg', 50, TRUE, -1)
+	target.apply_damage(rand(15,25), STAMINA, affecting, armor_block)
+	target.Knockdown(4 SECONDS)
+	log_combat(user, target, "leg sweeped", name)
 	return TRUE
 
-/datum/martial_art/krav_maga/proc/quick_choke(mob/living/A, mob/living/D)//is actually lung punch
-	D.visible_message(span_warning("[A] pounds [D] on the chest!"), \
-					span_userdanger("Your chest is slammed by [A]! You can't breathe!"), span_hear("You hear a sickening sound of flesh hitting flesh!"), COMBAT_MESSAGE_RANGE, A)
-	to_chat(A, span_danger("You pound [D] on the chest!"))
-	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
-	if(D.losebreath <= 10)
-		D.losebreath = clamp(D.losebreath + 5, 0, 10)
-	D.adjustOxyLoss(10)
-	log_combat(A, D, "quickchoked", name)
+/datum/martial_art/krav_maga/proc/neck_chop(mob/living/user, mob/living/target)
+	if(!iscarbon(target))
+		return FALSE
+	var/mob/living/carbon/carbon_defender = target
+	target.visible_message(span_warning("[user] karate chops [target]'s neck!"), \
+					span_userdanger("Your neck is karate chopped by [user], rendering you unable to speak!"), span_hear("You hear a sickening sound of flesh hitting flesh!"), COMBAT_MESSAGE_RANGE, user)
+	to_chat(user, span_danger("You karate chop [target]'s neck, rendering [target.p_them()] unable to speak!"))
+	playsound(get_turf(user), 'sound/effects/hit_punch.ogg', 50, 1, -1)
+	target.apply_damage(5, user.get_attack_type())
+	log_combat(user, target, "neck chopped", name)
+	carbon_defender.silent = 10 //10 life() ticks without the ability to speak
 	return TRUE
 
-/datum/martial_art/krav_maga/proc/neck_chop(mob/living/A, mob/living/D)
-	D.visible_message(span_warning("[A] karate chops [D]'s neck!"), \
-					span_userdanger("Your neck is karate chopped by [A], rendering you unable to speak!"), span_hear("You hear a sickening sound of flesh hitting flesh!"), COMBAT_MESSAGE_RANGE, A)
-	to_chat(A, span_danger("You karate chop [D]'s neck, rendering [D.p_them()] unable to speak!"))
-	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
-	D.apply_damage(5, A.get_attack_type())
-	if (iscarbon(D))
-		var/mob/living/carbon/carbon_defender = D
-		if(carbon_defender.silent <= 10)
-			carbon_defender.silent = clamp(carbon_defender.silent + 10, 0, 10)
-	log_combat(A, D, "neck chopped", name)
+
+/datum/martial_art/krav_maga/proc/martial_strikes(mob/living/user, mob/living/target)
+	if(!iscarbon(target))
```

Comment:
add text feedback to user

Reply:
What should the feedback say? Returning false makes an ordinary attack trigger instead of a martial art that is ineffective and appears to work.

Reply:
Is the martial strike being 'used up' upon hitting a noncarbon mob?

If this is the case, yes it should say to the user that the particular move is ineffective.

If martial strike is still queued, however, then theres no need to tell the user.

Reply:
It is indeed being used up, I've added error messages to both the neck chop and leg sweep.